clusterlib/scheduler.py

```diff
@@ -101,14 +101,15 @@ def _slurm_queued_or_running_jobs(user=None):
     try:
         with open(os.devnull, 'w') as shutup:
             out = subprocess.check_output(command, stderr=shutup)
-            out = out.splitlines()
+            out = out.decode(encoding)
+            out = [l for l in out.splitlines()]
             return out
     except OSError:
         # squeue is not available
         return []
 
 
-def queued_or_running_jobs(user=None):
+def queued_or_running_jobs(user=None, encoding='utf-8'):
```

Comment:
Can you add a docstring for the new parameter and explain the use case to the user?